meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -540,6 +541,7 @@ WKS_FILE_DEPENDS:remove:stm32mp15-disco = "st-image-bootfs st-image-userfs"
 WKS_FILE:stm32mp15-disco = "sdimage-stm32mp157c-dk2-optee.wks.in"
 WKS_FILE:sota:stm32mp15-disco = "sdimage-stm32mp157c-dk2-optee-sota.wks.in"
 LMP_BOOT_FIRMWARE_FILES:stm32mp15-disco = "arm-trusted-firmware/tf-a-stm32mp157c-dk2-sdcard.stm32 fip/fip-stm32mp157c-dk2-optee.bin"
+KERNEL_DEVICETREE:stm32mp15-disco = "stm32mp157c-dk2-scmi.dtb"
```

Comment:
This overrides the previous definition `KERNEL_DEVICETREE:stm32mp15-disco = "stm32mp157c-ev1-scmi.dtb"
` but look like the previous is for stm32mp15-eval